src/server/auth/auth.router.js

```diff
@@ -1,12 +1,25 @@
+// @flow
+
 import express from 'express';
 
+import type { Router, $Request, $Response } from 'express';
+
+import Authentication from './auth';
+import Database from '../db/database';
+
 export default class AuthRouter {
-  constructor(auth, database) {
+  router: Router<>;
+
+  auth: Authentication;
+
+  database: any;
```

Comment:
`any`? Isn't this an instance of your `Database` class?

Reply:
Yeah nice catch